### DISCUSSION TITLE
MultiSubnetFailover support

### DISCUSSION THREAD

**Discussion by LiviuTrifoi** [mkleehammer/pyodbc#1193] [Q&A]
I have a connection string that contains MultiSubnetFailover=Yes.    **Is this supported?**
I'm using pyodbc 4.0.35, python 3.8, linux. The client works without MultiSubnetFailover in connection string and connecting directly to the primary of the failover group.

When I try to connect, some connections work but some get random:
"pyodbc.OperationalError: ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Client unable to establish connection because an error was encountered during handshakes before login. Common causes include client attempting to connect to an unsupported version of SQL Server, server too busy to accept new connections or a resource limitation (memory or maximum allowed connections) on the server. (26) (SQLDriverConnect)')"

**Comment by v-chojas** [comment]
Yes, but this has nothing to do with pyODBC as it just passes through connection string to the driver.

**Comment by partiga-encoura** [comment]
I'm having the exactly same problem, any ideas on how to fix it?
